iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveTableUtil.java

```diff
@@ -225,4 +231,34 @@ public static boolean isCtas(Properties properties) {
     return Boolean.parseBoolean(properties.getProperty(hive_metastoreConstants.TABLE_IS_CTAS));
   }
 
+  protected static Properties getProps() {
```

Comment:
the method name is not self-explanatory, maybe rename it to `getSerializationProps()`?

Reply:
sure